README.md

```diff
@@ -58,37 +58,6 @@ Depending on your application add one of the following libraries as dependency t
 * classic (sync/blocking): `one.tomorrow.transactional-outbox:outbox-kafka-spring:$version`
 * reactive: `one.tomorrow.transactional-outbox:outbox-kafka-spring-reactive:$version`
 
-### Only for Spring Boot (sync/blocking): Add an OutboxSessionFactory
-
-Add a @Primary OutboxSessionFactory to map the EntityManager's session to Hibernate Session. Here's an example:
-
-```kotlin
-package one.tomorrow.seizure.kafka.publisher.config
-
-import one.tomorrow.transactionaloutbox.repository.OutboxSessionFactory
-import org.hibernate.Session
-import org.hibernate.SessionFactory
-import org.springframework.context.annotation.Primary
-import org.springframework.stereotype.Component
-import javax.persistence.EntityManager
-import javax.persistence.PersistenceContext
-
-@Component
-@Primary
-class SpringBootOutboxSessionFactory(
-
-    @PersistenceContext
-    private val entityManager: EntityManager,
-
-    private val sessionFactory: SessionFactory
-
-) : OutboxSessionFactory {
-
-    override fun getCurrentSession(): Session = entityManager.unwrap(Session::class.java)
-    override fun openSession(): Session = sessionFactory.openSession()
-}
-```
-
```

Comment:
There are other references to hibernate, which now should be changed to JPA I think.

Reply:
On a seconds view I'm not sure about that. AFAICS this module still depends on hibernate and cannot be used with other JPA implementations... Would be nice to get a PR description, what exactly the intention of this PR here is... 😉

Reply:
I've added a description, specifying the goal of the PR. Basically, as [Spring 6 doesn't support plain SessionFactory setups anymore](https://docs.spring.io/spring-framework/reference/data-access/orm/hibernate.html), we have to get rid of the SessionFactory and that's what this PR is aiming for. Based on my understanding, Spring doesn't force us to get rid of everything from Hibernate; that's why I think that, for example, the `OutboxRecord` annotations are still fine like they currently are.

Of course we could aim for removing the Hibernate dependency completely, but as this is not a 5-minute task regarding these annotations (it's probably not an insane amount of work, either, but still), I'd like to leave it out of this PR, keeping the focus on the SessionFactory/EntityManager change.

What do you think, @magro, @hnrkdmsk, @danielrehmann?

Reply:
Sounds good, makes sense! Because I brought that up I already resolve this topic - @hnrkdmsk / @danielrehmann just reopen if you feel the need for it.